presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -127,6 +127,12 @@ public static void setTypeInfo(Builder builder, TypeSignature type)
                 builder.setScale(0);
                 builder.setColumnDisplaySize(24);
                 break;
+            case "float":
+                builder.setSigned(true);
+                builder.setPrecision(7);
```

Comment:
Precision is 9 (not for all number though), e.g. `147483648f` is representable in IEEE754 float.